src/testdir/test_breakindent.vim

```diff
@@ -831,4 +831,22 @@ func Test_breakindent20_list()
   call s:close_windows('set breakindent& briopt& linebreak& list& listchars& showbreak&')
 endfunc
 
+" The following used to crash Vim. This is fixed by 8.2.3391.
+" This is a regression introduced by 8.2.2903.
```

Comment:
Just confirming that the test caused vim-8.2.3390 to crash and latest vim-8.2.3394 does not crash on Linux x86_64.
Thanks.